pyproject.toml

```diff
@@ -111,6 +111,8 @@ all = [
   "bentoml[io-image]",
   "bentoml[io-pandas]",
   "bentoml[grpc]",
+  "bentoml[grpc-reflection]",
+  "bentoml[grpc-channelz]",
```

Comment:
is it better to have all of grpc dependencies together? or will it be too large?

Reply:
better to separate out, similar to how we approach tracing dependencies.